inst/examples/3_ReportingWorkflow.R

```diff
@@ -37,7 +39,10 @@ lRaw <- list(
     rename(Status = site_status),
   Raw_STUDY = gsm.core::lSource$Raw_STUDY %>%
     rename(studyid = protocol_number) %>%
-    rename(Status = status)
+    rename(Status = status),
+  Raw_VISIT = gsm.core::lSource$Raw_VISIT %>%
+    mutate(visit_folder = foldername) %>%
```

Comment:
this is weird - issue filed to reconcile at some point: https://github.com/Gilead-BioStats/gsm.mapping/issues/57